partcad/src/partcad/converters/stl_converter.py

```
@@ -0,0 +1,100 @@
+
+def simplify_shape(shape: TopoDS_Shape) -> TopoDS_Shape:
+    """Simplify the given shape using BRepBuilderAPI_Copy."""
+    copier = BRepBuilderAPI_Copy(shape)
+    return copier.Shape()
+
+
+def dump_shape(shape: TopoDS_Shape):
+    """Dump the details of the given shape for debugging."""
+    dump = io.BytesIO()  # Use BytesIO instead of StringIO
+    BRepTools.Dump_s(shape, dump)
+    dump_value = dump.getvalue().decode('utf-8')  # Decode bytes to string
```

Ruff (0.8.2)

Local variable `dump_value` is assigned to but never used

Remove assignment to unused variable `dump_value`

(F841)